static/src/javascripts/projects/commercial/modules/comscore.spec.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import config from 'lib/config';
+import { init, _ } from './comscore';
+
+const getComscoreScripTags = (): NodeList<HTMLElement> =>
+    document.querySelectorAll('script#comscore');
+
+const isValidComscoreScriptTag = (tag: HTMLElement) =>
+    tag instanceof HTMLScriptElement &&
+    tag.src === _.comscoreSrc &&
+    tag.async === true;
+
+describe('comscore init', () => {
+    beforeEach(() => {
+        config.set('switches.comscore', true);
+    });
+
+    it('should do nothing if the comscore switch is off', () => {
+        config.set('switches.comscore', false);
+        init();
+
+        const comscoreTags = getComscoreScripTags();
+        expect(comscoreTags.length).toBe(0);
+    });
+
+    it('should drop the corect script tag if the comscore switch is on', () => {
+        config.set('switches.comscore', true);
+        init();
+
+        const comscoreTags = getComscoreScripTags();
+        expect(comscoreTags.length).toBe(1);
+        expect(isValidComscoreScriptTag(comscoreTags[0])).toBe(true);
+    });
+});
+
+describe('comscore getGlobals', () => {
+    it('returns an object with the correct comscorekw variable set', () => {
+        const randomStr = Math.random()
+            .toString(36)
+            .substring(2);
+
+        expect(_.getGlobals('')).toMatchObject({ comscorekw: '' });
+        expect(_.getGlobals(randomStr)).toMatchObject({
+            comscorekw: randomStr,
+        });
+    });
+
+    it('returns an object with no comscorekw variable set when called with "Network Front" as keywords', () => {
+        expect(_.getGlobals('Network Front')).toEqual(
+            // $FlowFixMe property not does actually exist
+            expect.not.objectContaining({ comscorekw: expect.any(Object) })
```

Comment:
Would this be more clearly expressed by taking the object keys and checking for `comscorekw`?

Reply:
👍